controller/attribute/manager.py

```diff
@@ -96,12 +97,10 @@ def update_attribute(
         source_code,
         with_commit=True,
     )
-    if attribute.get(project_id, attribute_id).state in [
-        AttributeState.UPLOADED.value,
-        AttributeState.AUTOMATICALLY_CREATED.value,
-        AttributeState.USABLE.value,
-    ]:
-        notification.send_organization_update(project_id, "attributes_updated")
+
+    notification.send_organization_update(
+        project_id=project_id, message=f"calculate_attribute:updated:{str(attribute_item.id)}"
+    )
```

Comment:
i think we will run into some merge issues here (since I had to change the websocket notification a bit as well.
-> it's not wrong, just a reminder for us so we don't forget it :)